geomstats/visualization.py

```diff
@@ -5,12 +5,12 @@
 import geomstats.backend as gs
 from geomstats.geometry.hyperboloid import Hyperboloid
 from geomstats.geometry.hypersphere import Hypersphere
-from geomstats.geometry.special_euclidean import SpecialEuclidean
-from geomstats.geometry.special_orthogonal import SpecialOrthogonal
+from geomstats.geometry.special_euclidean import SpecialEuclidean3
+from geomstats.geometry.special_orthogonal import SpecialOrthogonal3
 from mpl_toolkits.mplot3d import Axes3D  # NOQA
 
-SE3_GROUP = SpecialEuclidean(n=3)
-SO3_GROUP = SpecialOrthogonal(n=3)
+SE3_GROUP = SpecialEuclidean3()
+SO3_GROUP = SpecialOrthogonal3()
```

Comment:
Ditto: this syntax is definitely not user-friendly. Let's keep SpecialEuclidean(n=3) and modify SpecialEuclidean's constructor, to let it handle the n=3 and 'vector' case.